Add release notes for 0.9.8 (#2332)
scalameta/metals: website/blog/2020-12-18-lithium.md

---

We're happy to announce the release of Metals v0.9.8, which brings along a
number of smaller improvements, new Scala 3 milestone support and a hot new
feature.

<table>
<tbody>
  <tr>
    <td>Commits since last release</td>
    <td align="center">99</td>
  </tr>
  <tr>
    <td>Merged PRs</td>
    <td align="center">40</td>
  </tr>
    <tr>
    <td>Contributors</td>
    <td align="center">4</td>
  </tr>
  <tr>
    <td>Closed issues</td>
    <td align="center">5</td>
  </tr>
  <tr>
    <td>New features</td>
    <td align="center">1</td>
  </tr>
</tbody>
</table>

For full details: https://github.com/scalameta/metals/milestone/31?closed=1

Metals is a language server for Scala that works with VS Code, Vim, Emacs,
Sublime Text, Atom and Eclipse. Metals is developed at the
[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
with the help from [Lunatech](https://lunatech.com) along with contributors from
the community.

## TL;DR

Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
give Metals a try!

- Add in ability to export worksheet evaluations.
- Scala 3.0.0-M2 support.

## Worksheet evaluations

> Thanks to the great work by [ckipp01](https://github.com/ckipp01) users will be
> now able to export their evaluated worksheets using a new command
> `copy-worksheet-output` or `Metals: Copy worksheet output` in Visual Studio

Reviewer: ```suggestion
Thanks to the great work by [ckipp01](https://github.com/ckipp01) users will now be able to export their evaluated worksheets using a new command,
`copy-worksheet-output`, or `Metals: Copy worksheet output` in Visual Studio
```

Author: Changed

Code.

> ![worksheet-export](https://user-images.githubusercontent.com/13974112/102061269-8903b180-3df3-11eb-9c68-ecb080bdb249.gif)

Reviewer: Ha, this was just my lazy gif with one line in it to show it works. We may want to put a better one in here showing something more than a single line unless you think this is fine.

Author: I should probably do that, I was just a bit lazy while writing the release notes :sweat_smile:

Author: Added a new gif, should be ok now.

> When using this command Metals will copy the existing statements to your

Reviewer: ```suggestion
When using this command existing statements will be copied to your
```
I wonder if we should phrase it like this to avoid any confusion. The client needs to be the one to actually do this part, so maybe the is _technically_ more true by not saying "metals will copy". Probably not a huge deal though.

Author: Fixed!
